pkgs/data/fonts/jetbrains-mono/default.nix

```diff
@@ -1,20 +1,20 @@
 { lib, fetchzip }:
 
 let
-  version = "1.0.1";
+  version = "1.0.2";
 in
 fetchzip rec {
   name = "JetBrainsMono-${version}";
 
-  url = "https://github.com/JetBrains/JetBrainsMono/releases/download/${version}/JetBrainsMono-${version}.zip";
+  url = "https://github.com/JetBrains/JetBrainsMono/releases/download/v${version}/JetBrainsMono-${version}.zip";
 
-  sha256 = "15a8fwyg8ns6krq6nsvgn41iaqbd3lgm3cmv7w370gr6brbn6lxq";
+  sha256 = "0m0l3w7vyhad22fazvp3z3xvs12zqy4b6cfa3m4y0vdd2vzh4a4k";
 
   postFetch = ''
-    unzip $downloadedFile
-    install -m444 -Dt $out/share/fonts/truetype ttf/*.ttf
-    install -m444 -Dt $out/share/fonts/woff/ web/woff/*.woff
-    install -m444 -Dt $out/share/fonts/woff2/ web/woff2/*.woff2
+    mkdir -p $out/share/fonts
+    unzip -j $downloadedFile \*.ttf -d $out/share/fonts/truetype
+    unzip -j $downloadedFile \*.woff -d $out/share/fonts/woff
+    unzip -j $downloadedFile \*.woff -d $out/share/fonts/woff2
```

Comment:
I think it should be `\*.woff2` on the second line.

Reply:
Fixed, thanks!